pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -68,6 +68,11 @@ const (
 	errorStatus  = "ERROR"
 
 	ServiceAnnotationLoadBalancerFloatingNetworkId = "loadbalancer.openstack.org/floating-network-id"
+
+	// ServiceAnnotationLoadBalancerInternal is the annotation used on the service
+	// to indicate that we want an internal loadbalancer service.
+	// If the value of ServiceAnnotationLoadBalancerInternal is false, it indicates that we want an external loadbalancer service. Default to true.
+	ServiceAnnotationLoadBalancerInternal = "service.beta.kubernetes.io/openstack-load-balancer-internal"
```

Comment:
Sorry i think i should have been more specific, looking at line 70, can we use `loadbalancer.openstack.org/openstack-internal-load-balancer` instead of `service.beta.kubernetes.io/openstack-load-balancer-internal`